programs/zstdcli.c

```diff
@@ -147,7 +147,7 @@ static void usage(FILE* f, const char* programName)
     DISPLAY_F(f, "Options:\n");
     DISPLAY_F(f, "  -o OUTPUT                     Write output to a single file, OUTPUT.\n");
     DISPLAY_F(f, "  -k, --keep                    Preserve INPUT file(s). [Default] \n");
-    DISPLAY_F(f, "  --rm                          Remove INPUT file(s) after successful (de)compression.\n");
+    DISPLAY_F(f, "  --rm                          Remove INPUT file(s) after successful (de)compression. Ignoring `--rm` command when the output is `stdout`.\n");
```

Comment:
For `zstd -h`, we try to keep displayed lines short, for a good terminal experience.
So I would try to shorten this one if possible, for example: 
`Remove INPUT file(s) after successful (de)compression to file.`